nlp/start.py

```diff
@@ -0,0 +1,38 @@
+import os
+import subprocess
+import sys
+
+
+def run_uvicorn():
+    # Get port from environment variable, default to 8002 if not set
+    port = os.environ.get('PORT', '8002')
+
+    # Ensure port is a valid integer
+    try:
+        port = int(port)
+    except ValueError:
+        print(f"Error: PORT environment variable must be a valid integer. Got: {port}")
+        sys.exit(1)
```

Comment:
This part seems to me a little redundant. Because we don't have environment variable so it will never be found and value 8002 will be set. I think that if we don't want to change port we can hard code it.